WordPress/Classes/ViewRelated/Stats/Subscribers/StatsSubscribersStore.swift

```diff
@@ -48,6 +51,34 @@ struct StatsSubscribersStore: StatsSubscribersStoreProtocol {
             }
         }
     }
+
+    func updateChartSummary() {
+        guard chartSummary.value != .loading else { return }
+
+        let unit = StatsSubscribersSummaryData.Unit.day
+        let cacheKey = StatsSubscribersCache.CacheKey.chartSummary(unit: unit.rawValue, siteId: siteID)
+        let cachedData: StatsSubscribersSummaryData? = cache.getValue(key: cacheKey)
+
+        if let cachedData = cachedData {
+            self.chartSummary.send(.success(cachedData))
+        } else {
+            chartSummary.send(.loading)
+        }
+
+        statsService.getSubscribers(unit: unit) { result in
```

Comment:
As mentioned https://github.com/wordpress-mobile/WordPressKit-iOS/pull/795#discussion_r1577375152, it should be possible to use `statService.getData` here for the same effect.